spec/features/activity_page_spec.rb

```diff
@@ -77,17 +76,27 @@
 
         it 'displays a list of revisions suspected of plagiarism' do
           view_plagiarism_page
-          assert_page_content article.title.gsub('_', ' ')
+          assert_page_content article.title.tr('_', ' ')
```

Comment:
When I upgraded rubocop, it started throwing a warning about this, saying to replace gsub with tr. Apparently, it's a faster, less flexible string method: it only replaces a single character with another single character.